### DISCUSSION TITLE
Required Survey para cambio de estado en un Unbound SM

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#32] [2. Questions and Answers] [answered]
Hola!
Estoy intentando que para una maquina de estado que no cree canales asociados a la task, sea manejado el cambio de estado solamente por required survey a través de la taskview. Sin embargo, cuando intento poner unbound a la sm, no me aparece la opción de cambio de estado con required survey.
Por tanto, me gustaría saber cómo podría llevar a cabo con buenas prácticas que la task no tenga un canal asociado y hacer uso de required survey para cambio de estado.

**Comment by rafaelwalter** [comment, marked as answer]
@ValeMP RequiredSurvey solo soporta SMs con channel type bound

**Reply by acedward** [reply]
@ValeMP esto es un feature en aún en desarrollo. Estamos buscando unificar todas las herramientas.

**Reply by ValeMP** [reply]
okis, gracias! 
@acedward @rafaelwalter saben que podría hacer para lograr el efecto que quiero. Que no se logre ver el channel ? 
Voy pensando y se podría dejar el channel sin ningún participante adentro, así existe pero no se ve.
